cmd/cosign/cli/verify/verify.go

```diff
@@ -556,3 +510,69 @@ func shouldVerifySCT(ignoreSCT bool, keyRef string, sk bool) bool {
 	}
 	return true
 }
+
+// loadCertsKeylessVerification loads certificates for the verification of keyless signatures
+// for the verify command.
+//
+// TODO(dmitris) - mention additionally verify-attestation, verify-blob, verify-blob-attestation
```

Comment:
i would remove your username from here and just leave as a `TODO:`

Reply:
done in a8a73411, please re-thumb @cpanato